curriculum/dfe/content.json

```diff
@@ -6,7 +6,11 @@
   "config": {
     "placeholderText": "This is where users can type text",
     "defaultDocumentType": "personal",
-    "settings": {"table": {"numFormat": ".2~f"}, "text": {"tools": ["bold", "subscript", "superscript"]}},
```

Comment:
"settings" seems to be duplicated which I don't think works?

Reply:
I'm only seeing one copy of "settings" in there. What am I missing?

Reply:
ok, now it's gone. I swear to you there was one but I don't see it now so nevermind.